src/event_loop.rs

```diff
@@ -241,6 +243,16 @@ impl<T> EventLoopWindowTarget<T> {
     #[cfg(target_os = "windows")]
     self.p.set_device_event_filter(_filter);
   }
+
+  /// Returns the current cursor position
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **iOS / Android**: Unsupported.
+  #[inline]
+  pub fn cursor_position(&self) -> Result<PhysicalPosition<f64>, ExternalError> {
```

Comment:
I feel like this should be added in EventLoopWindowTarget instead. Otherwise, the usage is going to be pretty limited.